src/components/BackButton.js

```diff
@@ -16,6 +18,17 @@ class BackButton extends Component {
 		};
 	}
 
+	label() {
+		var label = "";
+		if (this.props.label == null) {
+			label = "Back";
+		}
+		if (this.props.label == "cancel") {
+			label = "Cancel";
+		}
+		return label;
+	}
```

Comment:
Rewrite as `label = () => this.props.label || "Back"` - short circuit logic allows you to concisely set a default as well as allow for more cases besides hardcoding. Use a `capitalize` function if needed.

Reply:
Really elegant solution